### DISCUSSION TITLE
Sacrifice of the Master

### DISCUSSION THREAD

**Discussion by Sol-au-fein** [secretcowlevel/DoomtrooperPublic#38] [Card Balance] [answered]
Sacrifice of the Master
Play at any time. For each Warrior you discard from your squad, you gain 10 [DP] .If played during COMBAT, noncombatants may be discarded in this manner.

Tax Refund
Play as 1 Action. Annihilate one of your Warriors in play to gain twice its modified [V]  in [DP]. Cannot play during COMBAT

Tax refund seems nicely balanced. It costs 1 action and annihilating 1 warrior. You can eventually get more than 10DP from it, but that is all.

Sacrifice to the Master on other hand is free to play, anytime, any number of warriors, discarding not annihilating and you get 10DP for each.

**Comment by Sol-au-fein** [comment, marked as answer]
Suggestion:
Gain 5DP for each.

**Reply by 21-ric** [reply]
I think it's a good suggestion. 
Currently there is no interest to play "tax refund" if you have "sacrifice of the master".

**Reply by Michalx11** [reply]
Or Use 1 Action per sacrifice plus ABSOLUTLY NOT ALLOWED TO BE PLAYED DURING COMBAT

**Reply by TomekPL-1** [reply]
I agree with Sol-au-fein and Michalx11 idea:
Sacrifice of the Master
Play at any time. For each Warrior you discard from your squad, you gain 5 [DP] . This card cannot be played during COMBAT.
